### PR TITLE
chore(weaver): commit prototypes to allow one-step build

### PR DESCRIPTION
**Pull request by jenniferlianne**
The cactus monorepo build requires local dependency pacakges to be built as a prerequisite.  This commit checks in prototype files generated by the weaver build process so that the forthcoming copm packages can be built in ci without a pre-build code generation step.

**Pull Request Requirements**
- [x] Rebased onto `upstream/main` branch and squashed into single commit to help maintainers review it more efficient and to avoid spaghetti git commit graphs that obfuscate which commit did exactly what change, when and, why.
- [x] Have git sign off at the end of commit message to avoid being marked red. You can add `-s` flag when using `git commit` command. You may refer to this [link](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) for more information.
- [x] Follow the Commit Linting specification. You may refer to this [link](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#specification) for more information. 

**Character Limit**
- [x] Pull Request Title and Commit Subject must not exceed 72 characters (including spaces and special characters).
- [x] Commit Message per line must not exceed 80 characters (including spaces and special characters).

**A Must Read for Beginners**
For rebasing and squashing, here's a [must read guide](https://github.com/servo/servo/wiki/Beginner's-guide-to-rebasing-and-squashing) for beginners.

### CODE REVIEW

File: weaver/sdks/fabric/interoperation-node-sdk/protos-js/common/access_control_pb.d.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+// package: common.access_control
```
Comment (sandeepnRES) [inline weaver/sdks/fabric/interoperation-node-sdk/protos-js/common/access_control_pb.d.ts:1]: We should not commit this `protos-js`, its a duplicate of the one in `weaver/common/protos-js`.
While building `sdk` using `make build-local`, the makefile will copy it here...

Comment (VRamakrishna) [reply weaver/sdks/fabric/interoperation-node-sdk/protos-js/common/access_control_pb.d.ts:1]: I agree. Having duplicates carries the risk of inconsistency between the sdk and the protos. If we are doing a local build, it's easy to rebuild the protos (if required) and copy here. For the regular build, there's a dependency to the protos package published to NPM anyway, so there's no need to fetch the built JS code.

**Review by VRamakrishna — CHANGES_REQUESTED** [review]
I suggest removing the new files under `sdk`. If you see a strong need to keep them, let's have a conversation on it.

**Comment by jenniferlianne** [comment]
Closing as no longer needed.

**Review by VRamakrishna — APPROVED** [review]

**Review by VRamakrishna — CHANGES_REQUESTED** [review]
Sorry, I approved without seeing your comment about closing this. Let me un-approve so this doesn't get merged by accident.

**Pull request closed without merging**